thefuck/rules/git_clone_missing.py

```diff
@@ -0,0 +1,26 @@
+from thefuck.specific.git import git_support
+
+output_contains = ': not found'
```

Comment:
Fish returns `repo.git : command not found`

Reply:
Base returns: `: No such file or directory`

Reply:
So you might consider searching for `not found` or `no such file or directory`?

Reply:
Fixed with refactor